Vision/README.md

```diff
@@ -11,39 +9,37 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
 
-Our new vision is built on these twin pillars:
-
-* Our existing pillar, on which we built the web: the enormous potential of technological change.
-* Our new pillar for the years ahead: recognition and embodiment of fundamental values into the architecture of the web.
-
-# Our Values for the world-wide web
-
-These are the core values of the Web:
+# Vision for the World-Wide Web
 
 * The Web is for all humanity.
 * The Web is designed for the good of its users.
 * The Web must be safe for its users.
 * There is one interoperable world-wide Web.
 
-# Our action as a Consortium
-
-We will continue to develop a world-wide web that
-* Works for everyone, striving for diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. We strongly emphasize accessibility, internationalization, privacy, security and diversity.
-* Puts the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-* Is trustworthy, by ensuring security and privacy for users. 
-* Does not favor centralization.
-* Implements a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.
-* Is based on standards that are openly developed with consensus of industry and key stakeholders, standards rooted in a strong royalty-free patent policy and open copyright licenses, focussed on interoperability and collective empowerment, and supported by open test suites.
-
-# The Operating Principles of the Consortium
-
-In our operations and membership, we will:
-*	Provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
-* Encourage incubation in new areas and industries with open platforms for discussion, collaboration and innovation, making it more structured and improving consensus-building among key stake holders.
-* Strive for the broadest participation, along axes including worldwide participation, diversity and inclusion, facilitating balance, equity and cooperation among the participants from different industries, user groups and organizational sizes, and thus establishing W3C as representative of the whole community.
-* Increase involvement of under-represented key stakeholders such as end users, content creators, developers etc.
-* Ensure transparency, equity and fairness. Our work will not be exclusively dominated by any person, company or interest group.
-* Establish and improve collaborative relationships with other organizations in the domain of Internet and Web standards, including building and maintaining respected relationships with governments and businesses for providing credible advice.
```

Comment:
These sections don't go away - all the items in them are present in either the Vision or the Principles and Values, EXCEPT "* Implements a unified, extensible, Web architecture, which continues to address evolving use cases for the general public."  (That seems out of place here, and more appropriately covered in the TAG's design principles.)